src/renderers/webgl/WebGLTextures.js

```diff
@@ -570,7 +570,7 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 		}
 
-		if ( extensions.has( 'EXT_texture_filter_anisotropic' ) === true ) {
+		if ( texture.magFilter !== NearestFilter && extensions.has( 'EXT_texture_filter_anisotropic' ) === true ) {
```

Comment:
Shouldn't you check for `minFilter` instead?

Reply:
Turns out both need to be checked, see response in thread below